syne_tune/tuner.py

```diff
@@ -682,4 +674,4 @@ def _default_callback():
         """
         :return: Default callback to store results
         """
-        return StoreResultsCallback()
+        return StoreResultsCallback()
```

Comment:
This is just adding a newline after https://github.com/awslabs/syne-tune/blob/main/syne_tune/tuner.py#L685 from mainline